examples/tensorflow/segmentation/train.py

```diff
@@ -50,6 +50,15 @@ def get_argument_parser():
                                         cpu_only=False,
                                         metrics_dump=False)
 
+    parser.add_argument(
```

Comment:
`train.py` does not require the mode train by design. Please revert this change.

Reply:
Done